tests/converter/test_binary.py

```diff
@@ -18,7 +18,7 @@
 import unittest
 from os.path import join
 
-from scripts import generate_project as gp
+from blueoil.converter import generate_project as gp
```

Comment:
If tests in `dlk/tests` if unit test, it's preffer to put these test files in `tests/unit/converter`.

Reply:
oh, okey

Reply:
I suppose these are not unittest... it would be better to re-define unittests for converter. can we discuss those in another issue?

Reply:
@kchygoe I see. OK to put tests in `tests/converter`